src/index.html

```diff
@@ -15,8 +15,49 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+      class="header"
+      id="header-1"
+    >
+      <a href="#header-1">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li>
+            <a
+              class="is-active"
+              href="#link1"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="#link2">Samsung</a></li>
+          <li><a href="#link3">Smartphones</a></li>
+          <li>
+            <a
+              data-qa="hover"
+              href="#link4"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li><a href="#link5">Gadgets</a></li>
+          <li><a href="#link6">Tablets</a></li>
+          <li><a href="#link7">Photo</a></li>
+          <li><a href="#link7">Video</a></li>
```

Comment:
The `href` attribute for the 'Photo' and 'Video' links are both set to `#link7`. This will cause both links to point to the same section, which is likely not the intended behavior. Consider using unique `href` values for each link to ensure they navigate to the correct sections.